schema/ismrmrd_example_extended.xml

```diff
@@ -376,6 +376,32 @@
     <TI>300.08</TI>
     <flipAngle_deg>90.0</flipAngle_deg>
     <flipAngle_deg>120.0</flipAngle_deg>
+    <diffusionDimension>segment</diffusionDimension>
+    <diffusion>
+      <gradientDirection>
+        <ap>1</ap>
+        <fh>0</fh>
+        <rl>0</rl>
+      </gradientDirection>
+      <bvalue>2.0</bvalue>
+    </diffusion>
+    <diffusion>
+      <gradientDirection>
+        <ap>0</ap>
+        <fh>1</fh>
+        <rl>0</rl>
+      </gradientDirection>
+      <bvalue>1.0</bvalue>
+    </diffusion>
+    <diffusion>
+      <gradientDirection>
+        <ap>0</ap>
+        <fh>0</fh>
+        <rl>1</rl>
+      </gradientDirection>
+      <bvalue>4.0</bvalue>
+    </diffusion>
+	  <diffusionScheme>bipolar</diffusionScheme>
```

Comment:
Can you please fix the spacing on this so it renders properly online?

Reply:
Sure. Should be fixed now.